pkg/portal/graphql/collaborator_mutation.go

```diff
@@ -49,3 +51,147 @@ var _ = registerMutationField(
 		},
 	},
 )
+
+var deleteCollaboratorInvitationInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "DeleteCollaboratorInvitationInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"collaboratorInvitationID": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.String),
+			Description: "Collaborator invitation ID.",
+		},
+	},
+})
+
+var deleteCollaboratorInvitationPayload = graphql.NewObject(graphql.ObjectConfig{
+	Name: "DeleteCollaboratorInvitationPayload",
+	Fields: graphql.Fields{
+		"app": &graphql.Field{Type: graphql.NewNonNull(nodeApp)},
+	},
+})
+
+var _ = registerMutationField(
+	"deleteCollaboratorInvitation",
+	&graphql.Field{
+		Description: "Delete collaborator invitation of target app.",
+		Type:        graphql.NewNonNull(deleteCollaboratorInvitationPayload),
+		Args: graphql.FieldConfigArgument{
+			"input": &graphql.ArgumentConfig{
+				Type: graphql.NewNonNull(deleteCollaboratorInvitationInput),
+			},
+		},
+		Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+			input := p.Args["input"].(map[string]interface{})
+			collaboratorInvitationID := input["collaboratorInvitationID"].(string)
+
+			gqlCtx := GQLContext(p.Context)
+			return gqlCtx.Collaborators.DeleteInvitation(collaboratorInvitationID).
+				Map(func(value interface{}) (interface{}, error) {
+					i := value.(*model.CollaboratorInvitation)
+					app := gqlCtx.Apps.Get(i.AppID)
+					return map[string]interface{}{
+						"app": app,
+					}, nil
+				}).Value, nil
+		},
+	},
+)
+
+var createCollaboratorInvitationInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "CreateCollaboratorInvitationInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"appID": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.ID),
+			Description: "Target app ID.",
+		},
+		"inviteeEmail": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.String),
+			Description: "Invitee email address.",
+		},
+	},
+})
+
+var createCollaboratorInvitationPayload = graphql.NewObject(graphql.ObjectConfig{
+	Name: "CreateCollaboratorInvitationPayload",
+	Fields: graphql.Fields{
+		"app": &graphql.Field{Type: graphql.NewNonNull(nodeApp)},
```

Comment:
I suppose we should include the full `CollaboratorInvitation` in the payload? Our convention is to include both the parent & created entity when creating, and only the parent entity when deleting.